src/ctx/RouteContext.ts

```
@@ -0,0 +1,173 @@
+	/**
+	 * @internal
+	 */
+	static useRouteHandler<TParams extends Params | undefined = Params, TData extends Data | undefined = Data>(
+		handler: RouteHandler<TParams, TData>,
+		params: Params,
+	): InternalHandler {
+		return (data, req) => {
+			// @ts-expect-error: InternalHandler ignores generics
+			const ctx = new RouteContext<TParams, TData>(req, data, params)
+			const run = async () => {
+				debugger;
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)